presto-hive-hadoop2/src/main/java/com/facebook/presto/hive/HiveHadoop2Plugin.java

```diff
@@ -13,11 +13,21 @@
  */
 package com.facebook.presto.hive;
 
+import com.facebook.presto.hive.metastore.ExtendedHiveMetastore;
+import com.google.inject.Module;
+
+import java.util.Optional;
+
 public class HiveHadoop2Plugin
         extends HivePlugin
 {
     public HiveHadoop2Plugin()
     {
         super("hive-hadoop2");
     }
+
+    public HiveHadoop2Plugin(String name, Optional<ExtendedHiveMetastore> metastore, Module pathProviderModule)
```

Comment:
`Optional<ExtendedHiveMetastore> metastore` is effectively unused.

Reply:
I removed changes to this plugin.